README.md

```diff
@@ -123,10 +123,12 @@ Besides basic filtering and aggregation, OpenSearch SQL also supports complex qu
 
 Recently we have been actively improving our query engine primarily for better correctness and extensibility. Behind the scene, the new enhanced engine has already supported both SQL and Piped Processing Language. Please find more details in [SQL Engine V2 - Release Notes](./docs/dev/NewSQLEngine.md).
 
-## Documentation
+## Documentation & Forum
```

Comment:
IMO, the Forum and Documentation should be two separate topics.  Why not just add a new section called "Discussion" or "Forum"?

Reply:
Can do.